pkg/adapter/nimbus-kubearmor/manager/manager.go

```diff
@@ -34,15 +34,15 @@ var (
 )
 
 func init() {
-	utilruntime.Must(intentv1.AddToScheme(scheme))
+	utilruntime.Must(v1alpha1.AddToScheme(scheme))
 	utilruntime.Must(kubearmorv1.AddToScheme(scheme))
 	k8sClient = k8s.NewOrDie(scheme)
 }
 
 func Run(ctx context.Context) {
 	npCh := make(chan common.Request)
 	deletedNpCh := make(chan common.Request)
-	go globalwatcher.WatchNimbusPolicies(ctx, npCh, deletedNpCh)
+	go globalwatcher.WatchNimbusPolicies(ctx, npCh, deletedNpCh, "SecurityIntentBinding", "ClusterSecurityIntentBinding")
```

Comment:
For better maintainability, let's update the `WatchNimbusPolicies` function to watch for changes in SIB and CSIB CRs, rather than passing them as arguments.

```suggestion
	go globalwatcher.WatchNimbusPolicies(ctx, npCh, deletedNpCh)
```



And we're already watching for CSIB CRs.
https://github.com/5GSEC/nimbus/blob/51a0c5b618aefc68aa9554bae2ee32ac90e82f20/pkg/adapter/nimbus-kubearmor/manager/manager.go#L49

Reply:
Please note that WatchNimbusPolicies is not watching for CISB. The function is watching for those NimbusPolicies whose owners are SIB and CSIB